### ISSUE TITLE
Error in iconv(enc2utf8(x), from = "UTF8", to = "cp65001") when using expss_output_viewer()

### ISSUE THREAD

**Issue by jpmyers2** [gdemin/expss#5]
Hi! 

Thank you for this great package!  

I am getting an error when using `expss_output_viewer()` or `expss_output_rnotebook()`.  **I am also able to reproduce this on a colleague's macOS machine**, but have successfully run this code on a Windows machine.

**Code:**
```R
library(expss)
data(mtcars)
expss_output_viewer()
cro_cpct(mtcars$cyl, list(total(), mtcars$am, mtcars$vs))
```

**Error:**
```R
Error in iconv(enc2utf8(x), from = "UTF8", to = "cp65001") : 
  unsupported conversion from 'UTF-8' to 'cp65001'
```

I am using Rstudio Version 1.1.419 on macOS Version 10.12.6 

Note, the `expss_output_default()` outputs correctly to the console.  

Thank you!

**Comment by gdemin** [comment]
Hi!
As a quick fix try the following code:
```
library(expss)
data(mtcars)
options(expss.fix_encoding = FALSE)  
expss_output_viewer()
cro_cpct(mtcars$cyl, list(total(), mtcars$am, mtcars$vs))
```

**Comment by jp-myers** [comment]
Thank you for the quick reply - that fix works!

**Comment by gdemin** [comment]
Fixed in v0.8.7
